app/config/parameters.yml.dist

```diff
@@ -11,8 +11,6 @@ parameters:
     # database_password: %env.database_password%
 
     database_driver: pdo_mysql
-    database_driver_class: ~
```

Comment:
Don't forget to put in the release notes advice to admins to delete this line.

Reply:
Yup.
But it’s not really a problem to let that value une the file as it’s not used anymore.
I’ll also update the doc